src/bootstrap/config.rs

```diff
@@ -200,6 +200,9 @@ pub struct Config {
     pub initial_rustc: PathBuf,
     pub initial_rustfmt: Option<PathBuf>,
     pub out: PathBuf,
+
+    /// Clippy Lint Level: Allow, Deny, Warn, Forbid
+    pub clippy_lint_level: [Option<String>; 4],
```

Comment:
We don't group options like this anywhere else; I don't think we should do it here. If we want to avoid extra fields I'd be fine with this as clippy_args, though.

Reply:
So, you want something like this field: `clippy_lint_*`. For every -a, -w, -d and -f?

Reply:
Yes, as I believe is already the case for flags.rs?

Reply:
done